src/actions/users.js

```diff
@@ -224,6 +224,8 @@ export function getProfiles(offset, limit = General.PROFILE_CHUNK_SIZE) {
             return null;
         }
 
+        const profiles = profileListToMap(profilesList);
```

Comment:
yeah I think its ok having a map in the store and the API can return whatever, we don't need to match the API or the store to the same object model, actually I think that was a mistake in APIv3